lib/fs.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+async function mkdirp(pathName, callback) {
```

Comment:
We don't need `async/await` here. What @nechaido was talking [here](https://github.com/metarhia/common/pull/249#discussion_r254176051) about is asynchronous function i.e. a function with the callback.